### ISSUE TITLE
Shift + arrow to select multiple values

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#8]
Holding shift and using the arrows (up/down) should select multiple values. When moving to a new line, the cell value should be saved to `selState.rows`. First when releasing the shift button, the selections are sent to engine. Going back the other way does not deselect the values.

Note that this has low prio and is not part of MVP

**Comment by LiKang6688** [comment]
https://help.qlik.com/en-US/sense/February2022/Subsystems/Hub/Content/Sense_Hub/UserInterface/keyboard-navigation-and-shortcuts.htm

**Comment by haxxmaxx** [comment]
Moved to different task manager
